### DISCUSSION TITLE
Zephyr and mcuboot

### DISCUSSION THREAD

**Discussion by rockoguido** [polarfire-soc/polarfire-soc-documentation#309] [RTOS for PolarFire SoC]
Zephyr is distributed with a bootloader, mcuboot.    Has a project been done to show mcuboot booting Zephyr

Thanks!
Dave

**Comment by con-pax** [comment]
Hi @rockoguido,

We suggest using the Hart Software Services (HSS) in combination with the HSS payload generator as the bootloader for a Zephyr application. The HSS is highly configurable and can be optimized/ tailored for a specific application. No formal work has been done on MCUBoot and Zephyr w.r.t PolarFire SoC...yet :-)
